### PR TITLE
feat: rename devicename and profilename

### PR DESCRIPTION
**Pull request by vli11**
Signed-off-by: Valina Li <valina.li@intel.com>

### CODE REVIEW

**Comment by vli11** [comment]
@lenny-intel  this is the first draft of rename devicename and profile name, I will make other files change if new names are good

File: loss-detection-app/res/device-rest/profiles/pos.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Copyright © 2022 Intel Corporation. All rights reserved.
 # SPDX-License-Identifier: BSD-3-Clause
 
-name: "device-pos-rest"
```
Comment (lenny-intel) [inline loss-detection-app/res/device-rest/profiles/pos.yaml:4]: Remove this file as it is handled by device-mqtt

File: loss-detection-app/res/device-rest/profiles/rfid-roi.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Copyright © 2022 Intel Corporation. All rights reserved.
 # SPDX-License-Identifier: BSD-3-Clause
 
-name: "device-rfid-roi-mqtt"
+name: "rfid-roi"
```
Comment (lenny-intel) [inline loss-detection-app/res/device-rest/profiles/rfid-roi.yaml:4]: Remove this file as it is handled by device-mqtt

File: loss-detection-app/res/device-rest/profiles/scale.yaml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # Copyright © 2022 Intel Corporation. All rights reserved.
 # SPDX-License-Identifier: BSD-3-Clause
 
-name: "device-scale-rest"
+name: "scale"
```
Comment (lenny-intel) [inline loss-detection-app/res/device-rest/profiles/scale.yaml:4]: Remove this file as it is handled by device-mqtt

**Review by lenny-intel — CHANGES_REQUESTED** [review]

**Review by jim-wang-intel — APPROVED** [review]
LGTM

**Review by marcpfuller — APPROVED** [review]
LGTM

**Pull request merged**